### PR TITLE
[Banner] Add snapshot tests for an orientation case.

### PR DESCRIPTION
**Pull request by fluxminds**
This test follows up the comment on https://github.com/material-components/material-components-ios/pull/8752#pullrequestreview-313547178.

The screenshot shows the change before and after https://github.com/material-components/material-components-ios/pull/8752.

Screen Shot 2019-11-07 at 3 40 37 PM

### CODE REVIEW

File: components/Banner/tests/snapshot/MDCBannerSnapshotTests.m [hunk 1]
```diff
@@ -235,6 +239,25 @@ - (void)testLongTextWithTwoActionsLTR {
   [self generateSnapshotAndVerifyForView:self.bannerView];
 }
 
+- (void)testLongTextWithTwoActionsWithTwoOrientationChangeActions {
+  // When
```
Comment (romoore) [inline components/Banner/tests/snapshot/MDCBannerSnapshotTests.m:243]: If this is an iOS 13 snapshot, why does it not guard against execution on early XCode/iOS versions?  From the snapshot code, it looks like specifying "YES" on an earlier device will result in the test silently not executing.

```
- (void)snapshotVerifyView:(UIView *)view
                 tolerance:(CGFloat)tolerancePercent
              supportIOS13:(BOOL)supportIOS13 {
  if (!supportIOS13 && ![self isSupportedDevice]) {
    return;
  } else if (supportIOS13 && ![self isSupportedIOS13Device]) {
    return;
  }
```

Would you please add a comment to either the test method or the "Then" section to indicate why this test should not run on earlier iOS versions?

**Review by romoore — APPROVED** [review]
LGTM with concerns about the test not executing on most iOS versions.

**Comment by fluxminds** [comment]
Blocked on cocoapods failure of "Timed out waiting for confirmation of orientation change".

File: components/Banner/tests/snapshot/MDCBannerSnapshotTests.m (hunk 1 above)
Comment (fluxminds) [reply components/Banner/tests/snapshot/MDCBannerSnapshotTests.m:243]: Good points! Will do.

**Comment by romoore** [comment]
Cocoapods tests aren't required. Did you mean "kokoro"?

**Comment by fluxminds** [comment]
Kokoro works fine. Cocoapods complains about it. Seems a known issue when I googled it. 

If it is not required, I probably will skip it after double checking, though I probably don't want to put an unstable test into the test suite.

**Comment by jverkoey** [comment]
Closing this PR due to inactivity, but please re-open it again if / when you intend to pursue it.

**Pull request closed without merging**